site/content/in-dev/unreleased/generic-table.md

```diff
@@ -0,0 +1,168 @@
+- **doc** (optional): Comment or description for the table
+
+## Generic Table API Vs. Iceberg Table API
+
+Generic Table provides a different set of APIs to operate on the Generic Table entities while Iceberg APIs operates on
+the Iceberg Table entities.
+
+| Operations   | **Generic Table API**                                       | **Iceberg Table API**                                       |
+|--------------|-------------------------------------------------------------|-------------------------------------------------------------|
+| Create Table | create an Iceberg table                                     | create a generic table                                      |
+| Load Table   | load an Iceberg table, load a generic table throws an error | load a generic table, load an Iceberg table throws an error |
+| Drop Table   | drop an Iceberg table, drop a generic table throws an error | drop a generic table, drop an Iceberg table throws an error |
+| List Table   | list all Iceberg tables                                     | list all generic tables                                     |
+
+Note that generic table shares the same namespace with Iceberg tables, the table name has to be unique under the same namespace.
+
+## Working with Generic Table
+
+There are two ways to work with Polaris Generic Tables today:
+1) Directly communicate with Polaris through REST API calls using curl. Details will be described in the later section.
+2) Use the Spark Client provided if you are working with Spark. Please refer to [Polaris Spark Client]({{% ref "polaris-spark-client" %}}) for detailed instructions.
+
+### Create a Generic Table
+
+To create a generic table, you need to provide the corresponding fields as described in [What is a Generic Table](#what-is-a-generic-table).
+Following is the REST API for creating a generic Table:
+
+```json
+POST /polaris/v1/{prefix}/namespaces/{namespace}/generic-tables
+{
+  "name": "<table_name>",
+  "format": "<table_format>",
+  "base-location": "<table_base_location>",
+  "doc": "<comment or description for table>",
+  "properties": {
+    "<property-key>": "<property-value>"
+  }
+}
+```
+
+Here is an example to create a generic table with name `delta_table` and format as `delta` under a namespace `delta_ns` 
+for catalog `delta_catalog`:
+
+```json
+POST /polaris/v1/delta_catalog/namespaces/delta_ns/generic-tables
+{
+  "name": "delta_table",
+  "format": "delta",
+}
+```
+
+### Load a Generic Table
+The REST API for load a generic table is the following:
+
+```json
+GET /polaris/v1/{prefix}/namespaces/{namespace}/generic-tables/{generic-table}
+```
+
+Here is an example to load the table `delta_table`:
+```json
+GET /polaris/v1/delta_catalog/namespaces/delta_ns/generic-tables/{generic-table}
+```
+And the response looks like the following:
+```json
+{
+  "table": {
+    "name": "delta_table",
+    "format": "delta",
+    "base-location": null,
+    "doc": null,
+    "properties": null
```

Comment:
This example is rather confusing IMHO. It provides neither the location nor properties... What does it represent?

Reply:
This looks like a minimal generic table, with no optional fields set? But it's not a great example, since the table created in the above section had optional fields populated.

Reply:
This example shows that there is a Delta table named `delta_table`, but what can the reader/user do with this information other than avoid creating other tables with the same name?